smart_card_connector_app/src/background.js

```diff
@@ -98,9 +104,13 @@ function launchedListener() {
 }
 
 function openWindow() {
-  var data = {'clientAppListUpdateSubscriber':
-      messageChannelPool.addOnUpdateListener.bind(messageChannelPool),
-      'readerTracker': readerTracker};
```

Comment:
I missed this when reviewing the original code, but passing class instances between windows is unsafe, given the presence of compiler optimizations.